src/main/java/org/reaktivity/nukleus/amqp/internal/stream/AmqpServerFactory.java

```diff
@@ -265,6 +275,46 @@
     private final Array8FW.Builder<AmqpSymbolFW.Builder, AmqpSymbolFW> annonymousRW =
         new Array8FW.Builder<>(new AmqpSymbolFW.Builder(), new AmqpSymbolFW());
 
+    private final AmqpPerformativeTypeFW openPerformativeType = new AmqpPerformativeTypeFW.Builder()
+        .wrap(new UnsafeBuffer(new byte[3]), 0, 3)
+        .set(OPEN)
+        .build();
```

Comment:
Suggest we rename this to `openType` for readability, similar feedback for other performative types.